cmd/shfmt/Dockerfile

```diff
@@ -1,9 +1,13 @@
-FROM golang:1.13.5-alpine3.10
+FROM golang:1.13.5-alpine3.11 as build
 
 WORKDIR /src
 COPY . .
 RUN CGO_ENABLED=0 go build -ldflags '-w -s -extldflags "-static"' ./cmd/shfmt
 
-FROM busybox:1.31.1-musl
-COPY --from=0 /src/shfmt /bin/shfmt
+FROM alpine:3.11 AS alpine
```

Comment:
why add the 'AS' keyword if your not reusing it?

Reply:
Because you can specify it from the docker build command line with `--target alpine` and the CI actually does that when building the docker images.

Reply:
learned something new today :) Didn't know you could have multiple targets in a single file, that's pretty cool and saves me from needing multiple dockerfiles!